### PR TITLE
Renaming the current file and adding changes

### PR DESCRIPTION
**Pull request by M-Moghazi**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1538)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: arabic/Greetings.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+...السلام عليكم و رحمة الله و بركاته...
```
Comment (casey-collab) [inline arabic/Greetings.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline arabic/Greetings.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**